### DISCUSSION TITLE
Add new color scheme

### DISCUSSION THREAD

**Discussion by azzamsa** [almarklein/timetagger#477] [Ideas]
Hi

The current dark theme is little bit unreadable to my taste. Can we add a handful of color scheme?

![image](https://github.com/almarklein/timetagger/assets/17734314/13f7e688-f01c-4931-9254-6a06e31eb306)

Thank you for timetagger 🍜

**Comment by almarklein** [comment]
The dark theme has not received as much attention as the light theme, in part because I don't use it myself. This is something I could use some help with by someone who cares about a good dark theme :)

**Reply by azzamsa** [reply]
Should I add additional color schemes or improve the current dark theme using `https://coolors.co/contrast-checker` and other readily available color schemes?

**Reply by almarklein** [reply]
I think improving the current dark theme is great! If you want to design a theme that deviates a lot, then perhaps a new theme makes more sense.
